fares-data-build-tool/create-data
# refactor: cfd-587 Monorepo tidy up

# Description

Monorepo tidy up
Renaming of `shared` to `fdbt-types` and treating it as a shared local module
Prettier run for site, netex-output and exporter

# Testing instructions

- ensure fdbt-site builds and deploys
- ensure export builds and deploys
- ensure fdbt-netex-output builds and deploys

# Type of change
